repo-validation/test-repo/LICENSE.md

```
@@ -0,0 +1,23 @@
+# Custom License Template
+
+Copyright (c) <no value> <no value>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: no

(MD033, no-inline-html)